cvpy/base/ImageTable.py

```diff
@@ -19,7 +19,7 @@
 
 from swat import CASTable, CAS
 
-from cvpy.base.ImageType import ImageType
+from cvpy.base.ImageDataType import ImageDataType
```

Comment:
This should be ImageType and not ImageDataType. 

I missed adding ImageType.py file in my push. I will push it after we merge your PR. Can you revert to ImageType in all places where you changed it to ImageDataType?

Reply:
I think we need to change it to 'ImageDataType' in order to run our tests locally.

Reply:
I added ImageType.py to the refactor branch. Please "sync" your refactor branch on GitHub.com. Then pull/rebase your local refactor branch.

Reply:
Thanks!